apps/api/src/services/trade-simulator.service.ts

```diff
@@ -166,16 +166,20 @@ export class TradeSimulator {
         fromTokenChain,
         fromTokenSpecificChain,
       );
+
       const toPrice = await this.priceTracker.getPrice(
         toToken,
         toTokenChain,
         toTokenSpecificChain,
       );
 
-      console.log(`[TradeSimulator] Got prices:
-        From Token (${fromToken}): $${fromPrice} (${fromTokenChain})
-        To Token (${toToken}): $${toPrice} (${toTokenChain})
-    `);
+      console.log("[TradeSimulator] Got prices:");
+      console.log(
+        `  From Token (${fromToken}): ${JSON.stringify(fromPrice, null, 4)} (${fromTokenChain})`,
+      );
+      console.log(
```

Comment:
I moved this to three log calls because the multi-line string was not being displayed correctly.
I _think_ this has to do with the way stdout of the child process is piped into the parent stdout, but regardless this works for me.  

Also, `fromPrice` and `toPrice` are Objects so they were displayed as `[object Object]`.  I changed this so they are stringified.